### ISSUE TITLE
messes up theme layout

### ISSUE THREAD

**Issue by jonathanwichter** [karrak1/PHP-Fusion_Addons#1]
This is what my site looks like before I infuse it:
<img width="2279" alt="screen shot 2018-02-24 at 7 32 53 pm" src="https://user-images.githubusercontent.com/35083828/36636704-b4791be2-1999-11e8-8782-0e60fb942c74.png">

I infused your infusion I get this on my main page.
<img width="2274" alt="screen shot 2018-02-24 at 7 32 16 pm" src="https://user-images.githubusercontent.com/35083828/36636696-951fdc36-1999-11e8-9698-0a5397d900fe.png">
Hopefully it is an easy fix.

**Comment by jonathanwichter** [comment]
Is their a fix?

**Comment by karrak1** [comment]
works under 9.0 and 9.03, administration has not yet watched..

**Comment by jonathanwichter** [comment]
I have 9.0, but what do you mean admin not yet watched? How do I fix the the theme or css layout problem?

**Comment by karrak1** [comment]
theme fix 
/infusions/pm_control_panel/templates.php
9.03 version == if (version_compare(9.0, fusion_get_settings('version'), ">")) {}
9.0 version else {}

Administration checked under 9.03

**Comment by jonathanwichter** [comment]
So what your trying to say is that I can't use your infusion? I am new to the coding of v9 so give me in layman's terms.

**Comment by jonathanwichter** [comment]
where do I put that?

**Comment by karrak1** [comment]
You can use it under 9.0.

**Comment by jonathanwichter** [comment]
I understand that but where do I put it in that file to make it work.  Because when I infuse it messes up the layout of that website.
